seedbuilder/areas.ori

```diff
@@ -1793,16 +1793,24 @@ home: MoonGrottoAboveTeleporter
 	pickup: LeftGrottoTeleporterExp
 		casual-core WallJump DoubleJump
 		casual-core Climb DoubleJump ChargeJump
+		standard-abilities DoubleJump Dash Ability=3
 		expert-core Grenade Bash
 		expert-core Climb ChargeJump
+		expert-core DoubleJump Dash
 		expert-dboost ChargeJump Health=4
 		expert-dboost WallJump Glide Health=4
 		expert-dboost Climb Glide Health=4
```

Comment:
I suspect this path can be moved to `standard-dboost`. Just hold left, glide, and up - after hitting the spikes you'll safely reach the XP as long as you had sufficient initial height, and it's honestly pretty forgiving.